application/src/test/resources/standalone/config/router-config.json

```diff
@@ -75,7 +75,8 @@
       "reluctance": 4.0,
       "stairsReluctance": 1.65,
       "boardCost": 600,
-      "escalatorReluctance": 1.5
+      "escalatorReluctance": 1.5,
+      "escalatorSpeed": 0.45
```

Comment:
Sorry, this is coming quite late and we've been here before but. Should we move this into a separate config?

```json
{
  "walk": {
     "escalator": {
       "reluctance": 2,
       "speed": 0.45
    }
   } 
}
```

Reply:
If we agree that we want it, then we can also do it in a follow-up.

Reply:
Joel also suggested that, but I think that if you have just two prefs, it doesn't make sense to make a container yet. For three and above, yes. Note that WalkPreferences also has stairsReluctance and stairsTimeFactor, also two.

Reply:
Okay then.

Reply:
@t2gran also preferred separate config so we decided to do it.